quality_control_stock_oca/models/stock_picking.py

```diff
@@ -80,5 +80,25 @@ def _action_done(self):
                     )
                 )
             for trigger_line in _filter_trigger_lines(trigger_lines):
-                inspection_model._make_inspection(operation, trigger_line)
+                inspection = inspection_model._make_inspection(operation, trigger_line)
```

Comment:
Have you tought inheriting the _make_inspection() function with this code?

Reply:
Sorry, but I don't know what you mean. What's your suggestion for improving the _make_inspection method?

Reply:
The idea is putting the new lot of the inspection inside the _make_inspection() function instead of the _action_done() function. 

i think it makes more sense and reduces the complexity of the _action_done inheritance

However it is a small detail, I approve the PR